pkg/stdlib/pkgs/multicluster.cue

```diff
@@ -24,6 +24,7 @@
 		properties: {...}
 	}]
 	externalRevision?: string
+	dependsOn?: [...string]
```

Comment:
`Inputs` & `Outputs` can not be used in `deploy2env` for the same reason.
Maybe you can just add `...` here for more parameters, but this may cause some confusion.

/cc @Somefive

Reply:
It is ok to add dependsOn here. The `deploy2env` does not support dependsOn but we are planning to fix it in the `deploy` WorkflowStep.

Reply:
Adding `...` also works. I can accept both solution.